examples/keras_simple.py

```diff
@@ -0,0 +1,93 @@
+"""
+Optuna example that optimizes a neural network classifier configuration for the
+MNIST dataset using Keras.
+
+In this example, we optimize the validation accuracy of MNIST classification using
+Keras. We optimize the filter and kernel size, kernel stride and layer activation.
+
+We have following two ways to execute this example:
+
+(1) Execute this code directly.
+    $ python keras_simple.py
+
+
+(2) Execute through CLI.
+    $ STUDY_NAME=`optuna create-study --direction maximize --storage sqlite:///example.db`
+    $ optuna study optimize keras_simple.py objective --n-trials=100 --study $STUDY_NAME \
+      --storage sqlite:///example.db
+
+"""
+
+from keras.backend import clear_session
+from keras.datasets import mnist
+from keras.layers import Conv2D
+from keras.layers import Dense
+from keras.layers import Flatten
+from keras.models import Sequential
+from keras.optimizers import RMSprop
+
+import optuna
+
+
+N_TRAIN_EXAMPLES = 3000
+N_TEST_EXAMPLES = 1000
+BATCHSIZE = 128
+CLASSES = 10
+EPOCHS = 10
+
+
+def objective(trial):
+    # Clear clutter from previous Keras session graphs.
+    clear_session()
+
+    (x_train, y_train), (x_test, y_test) = mnist.load_data()
+    img_x, img_y = x_train.shape[1], x_train.shape[2]
+    x_train = x_train.reshape(-1, img_x, img_y, 1)[:N_TRAIN_EXAMPLES].astype('float32') / 255
+    x_test = x_test.reshape(-1, img_x, img_y, 1)[:N_TEST_EXAMPLES].astype('float32') / 255
+    y_train = y_train[:N_TRAIN_EXAMPLES]
+    y_test = y_train[:N_TEST_EXAMPLES]
```

Comment:
The second `y_train` in the right side seems to be a typo for `y_test`.

```python
- y_test = y_train[:N_TEST_EXAMPLES]
+ y_test = y_test[:N_TEST_EXAMPLES]
```

Reply:
Yikes ! Sorry about that ! That should be all fixed for you now